### DISCUSSION TITLE
Hosting HTML pages like PDFs on a WikiJS Page

### DISCUSSION THREAD

**Discussion by l-ucky** [requarks/wiki#5435] [Help / Questions]
OS: Debian - Docker and Docker
Wiki.js version: 2.5.276
Database engine: PostgreSQL 11-Alpine

Hello, I am trying to figure out a solution to hosting files on a wikipage relevant to writing articles and citing held copies of papers. Right now I can upload a PDF asset to WikiJS and then import the PDF as a link with MarkDown formatting and click the link which shows the PDF. How can I do this with HTML files? I have some I would like to host and every time I click the link to the HTML asset I uploaded into the WikiJS page it brings me to an URL like `wiki.website.com/name-of-the-file.html` which prompts me to make a new page. All I want is the html file to display like the PDF does.

**Comment by Astrak00** [comment]
Did you find any solution?

**Reply by l-ucky** [reply]
I tried different software and decided to handwrite an HTML website instead. It has a nice 90s-00s aesthetic that grew on me. In HTML you can just use the `<embed>` tag and reference the file in the same directory as the `index.html` file. It's way lighter and simpler to run than dedicated software.

**Comment by udik-chudik** [comment]
You can add following at the bottom of your **config.yml**:
```yaml
uploads:
  forceDownload: false

pageExtensions:
  - md
  - txt
```
This will tell wiki.js not to associate .html files (enabled by default) as pages nor document to download. In my case I had to purge config in MySQL as well (look at setting table, change values there as well). After that I was able to view html pages uploaded as assets.
